styles/styles.css

```diff
@@ -20,20 +23,312 @@
     margin: 0;
 }
 
+:root {
+    --purple: #5722CD;
+    --blue: #4845E4;
+    --pink: #ED4385;
+    --black: #0C0C0C;
+    --light-purple: #F2F0FF;
+    --light-grey: #F3F6F9;
+    --grey: #7A7A7A;
+}
 
+body {
+    font-family: "Roboto", sans-serif;
+    font-weight: 300;
+    color: var(--black);
+    font-size: 20px;
+}
 
 /* ------------------------------- 2. Typography ----------------------------------- */
 
+h1, h2, h3, h4 {
+    font-family: Merriweather, serif;
+}
+
+h1, h2 {
+    font-size: 48px;
+    padding: 15px 0;
+}
+
+h3 {
+    color: var(--purple);
+    font-size: 32px;
+    padding: 15px 0;
+    line-height: 1.5;
+}
+
+h4 {
+    color: var(--purple);
+    font-size: 22px;
+}
+
+p {
+    margin: 10px 0;
+}
+
+button {
+    background-color: var(--pink);
+    color: #FFF;
+    text-transform: uppercase;
+    font-size: 14px;
+    border: none;
+    border-radius: 50px;
+    padding: 25px 50px;
+    margin: 30px 0;
+    box-shadow: 0 10px 10px rgba(0, 0, 0, 0.20);
+}
+
+
 /* ------------------------------- 3. Layout --------------------------------------- */
 
 /* --------------- 3.1 Reusables ----------------------- */
 
+.outer-container {
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+}
+
+.inner-container {
+    flex-basis: 1400px;
+    flex-shrink: 1;
+    flex-grow: 0;
+    margin: 0 20px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+
 /* --------------- 3.2 Other --------------------------- */
 
+a {
+    text-decoration: none;
+    color: var(--purple);
+}
+
+img {
+    display: block;
+    width: 100%;
+}
+
+footer {
+    background-color: var(--black);
+    color: #FFF;
+    text-align: center;
+    font-size: 14px;
+    padding: 0.7rem;
+}
+
 /* ------------------------------- 4. Areas ---------------------------------------- */
 
 /* --------------- 4.1 Header ------------------------- */
 
+header {
+    padding: 30px 0 120px 0;
+    background: rgb(72, 69, 228);
+    background: linear-gradient(150deg, rgba(72, 69, 228, 1) 0%, rgba(87, 34, 205, 1) 100%);
+    color: #FFF;
+    position: relative;
+}
+
+.header-navigation {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    width: 100%;
+}
+
+.header-navigation a {
+    color: #FFF;
+}
+
+.header-navigation ul {
+    flex-basis: 400px;
+    flex-grow: 0;
+    flex-shrink: 1;
+    list-style: none;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+.header-text {
+    font-size: 28px;
+    padding: 100px 0;
+    text-align: center;
+    max-width: 850px;
```

Comment:
Check even of deze in een row-of column container staat. Want als het een row is, kun je hier `flex-basis` van maken!